### PR TITLE
Update cartodb.js to v3.14.3

### PR DESCRIPTION
**Pull request by alonsogarciapablo**
All changes required to update cartodb.js to 3.14.3.

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3773/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3990/))

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cartodb-ui",
-  "version": "3.13.16",
+  "version": "3.14.3",
```
Comment (alonsogarciapablo) [inline package.json]: Why was this so outdated?

**Comment by alonsogarciapablo** [comment]
@xavijam @fdansv PTAL. Thanks!

**Comment by fdansv** [comment]
\ô/

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3997/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3998/))

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cartodb-ui",
-  "version": "3.13.17",
+  "version": "3.14.3",
```
Comment (alonsogarciapablo) [inline package.json]: Is this right?

**Comment by xavijam** [comment]
Oh! :S, go ahead without fear.

**Pull request merged**